netboot-services/tftp/Dockerfile

```diff
@@ -1,16 +1,23 @@
 FROM alpine:3.18 as bootfiles
-RUN apk add --no-cache bash gcc binutils make perl xz-libs xz-dev mtools git libc-dev
+RUN apk add --no-cache bash gcc binutils make perl xz-libs xz-dev mtools git libc-dev curl openssl coreutils
 WORKDIR /build
 # Clone iPXE
 RUN git clone git://git.ipxe.org/ipxe.git
 RUN sed -i 's/#undef\tDOWNLOAD_PROTO_HTTPS/#define\ DOWNLOAD_PROTO_HTTPS/' ipxe/src/config/general.h
+RUN sed -i 's/\/\/#define PING_CMD/#define PING_CMD/' ipxe/src/config/general.h
+RUN sed -i 's/\/\/#define NSLOOKUP_CMD/#define NSLOOKUP_CMD/' ipxe/src/config/general.h
+RUN export CERT=ca.pem,isrgrootx1.pem,lets-encrypt-r3.pem && export TRUST=ca.pem,isrgrootx1.pem,lets-encrypt-r3.pem
```

Comment:
I'd really like a inline Comment here, why we need to add let's-encrypt certificates. preferably with a link to the github issue, that you found. Because I assume, these cert chains are no longer needed once the github issue on the ipxe repository is fixed.

Reply:
added